model_execution/producer.py

```diff
@@ -0,0 +1,11 @@
+from time import sleep
```

Comment:
this shouldn't be separate for now .

Reply:
We will not be having this produce file as we improve this service so i added it separately to test. Should i change it to a single file?

Reply:
if this is only for testing can remove the file before merging this PR to master